test/modules/TestAnalyzer/TestResultReporter.cs

```diff
@@ -101,16 +101,46 @@ static ModuleMessagesReport GetReceivedMessagesReport(string moduleId, double to
                 }
             }
 
-            // check if last message is older
-            if (DateTime.Compare(lastMessageDateTime.AddMilliseconds(toleranceInMilliseconds), endDateTime) < 0)
+            bool areMessagesMissing = missingCounter > 0;
+            bool areLatestMessagesTooOld = DateTime.Compare(lastMessageDateTime.AddMilliseconds(toleranceInMilliseconds), endDateTime) < 0;
+            string missingMessagesStatus = $"Missing messages: {missingCounter}. ";
```

Comment:
Should remove the space at the end here; and add space when you construct status message below.